QUESTIONS.md

```diff
@@ -3,3 +3,11 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+- Can we have 10-10-10 challenges around using REACT (small CRUD exercise?)
+- What are some ways to better manage burnout and paralysis analysis?
+- Advice on how to maintain continuity of thought when working on a long-term project?
+- How to use npkill?
```

Comment:
Good question,i started it up and it took forever then I just closed it

Reply:
JR said he was going to go over this, correct?